TestFCWTAPI/UnitTest1.cs

```diff
@@ -44,5 +44,19 @@ public void TestFixOutputArray()
             float[][] outputArray = FCWTAPI.FixOutputArray(testArray, 3, 1, 2);
             Assert.AreEqual(4, outputArray.GetLength(0)); 
         }
+
+        [Test]
+        public void TestToTwoDArray()
```

Comment:
In light of previous comment, add second jagged array that would trigger the InvalidOperationException handling and an Assert statement that would ensure the error messaging is correct.